.github/workflows/cicd.yml

```
@@ -0,0 +1,56 @@
+
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
+
+        env:
+            ECR_REGISTRY: ${{ secrets.ECR_REGISTRY }}
+            ECR_REPOSITORY: ${{ secrets.ECR_REPOSITORY }}
+            IMAGE_TAG: ${{ secrets.IMAGE_TAG }}
+
+        steps:
+            -   name: Checkout code
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)